types/onecolor/onecolor-tests.ts

```diff
@@ -0,0 +1,58 @@
+// This contains sample code which tests the typings. This code does not run, but it is type-checked.
+
+import onecolor = require('onecolor');
+
+// Contrustor
```

Comment:
Obvious typo

Reply:
Thanks for your review @peterblazejewicz, and I've amended this typo in this [commit](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/41253/commits/271db78639557c73559e4686ec4fcf29fb892ea8)